<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图标颜色编辑器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>单个转换</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>批量转换</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="funcComboBox">
        <item>
         <property name="text">
          <string>替换某个固定色值（不带alpha）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>替换某个固定色值（带alpha）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>色值不为某个值都替换（只判断alpha）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>色值不为某个值都替换（带alpha）</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="srcPathLineEdit">
        <property name="placeholderText">
         <string>输入原始文件路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetPathLineEdit">
        <property name="placeholderText">
         <string>输出转换结果文件路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>把</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="srcColorLineEdit">
        <property name="placeholderText">
         <string>原始色值(16进制ARGB整数值)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>转换为</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetColorLineEdit">
        <property name="placeholderText">
         <string>替换色值(16进制ARGB整数值)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>替换的时候保留原始Alpha值</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="singleBtn">
        <property name="toolTip">
         <string>单个转换输入一个图标文件路径，输出也是</string>
        </property>
        <property name="text">
         <string>开始转换</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>提示：1.如果不指定输出文件路径，转换的图标保存在图标当前文件夹，并且命名为增加指定后缀的方式（默认后缀_convert）
      2.不同图标色值实现不一致，可能需要选择不一样的替换方式才能更准确 (鼠标滚轮可以缩放视图，视图间有个滑块可以拖动大小)
      3.按住Ctrl键，可以选中色值表中指定某些值，再点击右键，可以替换指定选中的色值 （每次色值修改都会自动保存到文件）</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>原始图标预览</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="srcIconWidget" native="true">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>转换后图标预览</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="resultIconWidget" native="true">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>图标色值表(ARGB)</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>764</width>
             <height>340</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="colorValueWidget" native="true">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
